### DISCUSSION TITLE
How can I sort the order of pull requests inserted into magit?

### DISCUSSION THREAD

**Discussion by ParetoOptimalDev** [magit/forge#618] [Support] [answered]
I see that `forge-topic-list-order` will do this for `forge-list-topics`, but that doesn't affect the order of the pull requests in `forge-insert-pullreqs`.

My context is that I'm trying to migrate from using the github web interface to forge and would like to begin by making everything the same order to minimize the things to learn to transition.

**Comment by tarsius** [comment, marked as answer]
Thanks for the report!

The problem was (lack of) cache invalidation.

You'll have to update Magit as well.

**Reply by ParetoOptimalDev** [reply]
Thank you very much for the quick fix!
